pkgs/by-name/op/open-webui/package.nix

```diff
@@ -0,0 +1,113 @@
+{ lib
+, buildNpmPackage
+, fetchFromGitHub
+, writeShellApplication
+, python3
+}:
+let
+  pname = "open-webui";
+  version = "0.1.124";
+
+  src = fetchFromGitHub {
+    owner = pname;
+    repo = pname;
```

Comment:
Please don't use this.

`repo = pname;` is nice for DRY but creates a binding that goes too far.

See further information about this here: https://github.com/nix-community/nixpkgs-lint/issues/21